lib/token-verifier/WP_Auth0_JwksFetcher.php

```diff
@@ -0,0 +1,76 @@
+<?php
```

Comment:
[See this diff](https://github.com/auth0/auth0-PHP/pull/401/files#diff-8397559345a7685d64dc018493eeb7a1) for changes from approved PHP SDK ones.